deploy/helm/values.yaml

```diff
@@ -0,0 +1,556 @@
+      <!-- IMPORTANT -->
+      <!--
+      #
+      # Template parameters available:
+      #
+      -->
+      <yandex>
+          <users>
+              <clickhouse_operator>
+                  <networks>
+                      <ip>127.0.0.1</ip>
+                  </networks>
+                  <profile>clickhouse_operator</profile>
+                  <quota>default</quota>
+              </clickhouse_operator>
+          </users>
+          <profiles>
+              <clickhouse_operator>
+                  <log_queries>0</log_queries>
+                  <skip_unavailable_shards>1</skip_unavailable_shards>
+                  <http_connection_timeout>10</http_connection_timeout>
+              </clickhouse_operator>
+          </profiles>
+      </yandex>
+    02-clickhouse-default-profile.xml: |
+      <!-- IMPORTANT -->
+      <!-- This file is auto-generated -->
+      <!-- Do not edit this file - all changes would be lost -->
+      <!-- Edit appropriate template in the following folder: -->
+      <!-- deploy/builder/templates-config -->
+      <!-- IMPORTANT -->
+      <yandex>
+        <profiles>
+          <default>
+            <log_queries>1</log_queries>
+            <connect_timeout_with_failover_ms>1000</connect_timeout_with_failover_ms>
+            <distributed_aggregation_memory_efficient>1</distributed_aggregation_memory_efficient>
+            <parallel_view_processing>1</parallel_view_processing>
+          </default>
+        </profiles>
+      </yandex>
+    03-database-ordinary.xml: |-
+      <!-- IMPORTANT -->
+      <!-- This file is auto-generated -->
+      <!-- Do not edit this file - all changes would be lost -->
+      <!-- Edit appropriate template in the following folder: -->
+      <!-- deploy/builder/templates-config -->
+      <!-- IMPORTANT -->
+      <!--  Remove it for ClickHouse versions before 20.4 -->
+      <yandex>
+          <profiles>
+              <default>
+                  <default_database_engine>Ordinary</default_database_engine>
+              </default>
+          </profiles>
+      </yandex>
+# additionalResources -- list of additional resources to create (are processed via `tpl` function)
+additionalResources: []
+#  - |
+#    apiVersion: v1
+#    kind: ConfigMap
+#    metadata:
+#      name: {{ include "altinity-clickhouse-operator.fullname" . }}-cm
+#      namespace: {{ .Release.Namespace }}
+#  - |
+#     apiVersion: v1
+#     kind: Secret
+#     metadata:
+#       name: {{ include "altinity-clickhouse-operator.fullname" . }}-s
+#       namespace: {{ .Release.Namespace }}
+#     stringData:
+#       mykey: my-value
+
+dashboards:
+  # dashboards.enabled -- provision grafana dashboards as secrets (can be synced by grafana dashboards sidecar https://github.com/grafana/helm-charts/blob/grafana-6.33.1/charts/grafana/values.yaml#L679 )
+  enabled: false
+  # dashboards.additionalLabels -- labels to add to a secret with dashboards
+  additionalLabels: {}
+  #  grafana_dashboard: "1"
```

Comment:
According to https://github.com/grafana/helm-charts/blob/grafana-6.33.1/charts/grafana/values.yaml#L679
this line should uncomment with empty value instead of "1"?